src/components/CounterPage/CounterContainer/index.js

```diff
@@ -0,0 +1,64 @@
+import { Component } from "react";
+
+import Counter from "../Counter";
+
+class CounterContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true
+    }
+  }
+
+  handleIncrease = () => {
+    this.setState(() => { return { countValue: this.state.countValue + 1 } });
+  };
+
+  handleDecrease = () => {
+
+    if (this.state.countValue <= 0) {
+      this.setState({ countValue: 0 })
+
+    } else {
+      this.setState(() => { return { countValue: this.state.countValue - 1 } });
+    }
+  };
```

Comment:
You can improve this if statement.

What the purpose of calling setState method if state for this moment equal to 0.
Rework your logic using just one if statement without else